drivers/arduino/RGB_LED_Matrix_Lib/src/RGBLEDMatrix.cpp

```diff
@@ -327,9 +345,45 @@ void stopScanning(void) {
 
 unsigned int RGBLEDMatrix::nextTimerInterval(void) const {
 	// Calculates the microseconds for each scan
-	int mulitplier = _scanPass*1.25;	
 	
-	return  5*mulitplier;
+	return  5*multiplier5microseconds( _scanPass );
+}
+
+
+#elif defined ( ESP8266 )
+
+//
+// On the ESP8266 boards, use the timer0 to drive scan timing
+// Use D5 (GPIO14) as CLK and D7 (CPIO13) as SER
+//
+
+void inline timer0InteruptHandler (void){
+	gSingleton->shiftOutCurrentRow();
+	// reload the timer
+ 	timer0_write(ESP.getCycleCount() + 84 * gSingleton->nextTimerInterval());
+  	// update scan row. Done outside of interrupt stoppage since execution time can
+  	// be inconsistent, which would lead to vary brightness in rows.
+  	gSingleton->incrementScanRow();
+}
+
+void RGBLEDMatrix::startScanning(void) {
+	this->setup();
+	
+	noInterrupts();
+	timer0_isr_init();
+	timer0_attachInterrupt(timer0InteruptHandler);
+ 	timer0_write(ESP.getCycleCount() + 84 * gSingleton->nextTimerInterval());
+	interrupts();
+}
+
+void stopScanning(void) {
+	timer0_detachInterrupt;
```

Comment:
Shouldn't this be a function call? That is:
```
	timer0_detachInterrupt();
```
Note the addition of the parenthesis.